qupath-gui-fx/src/main/java/qupath/lib/display/ChannelDisplayInfo.java

```diff
@@ -941,7 +941,7 @@ public String getName() {
 		}
 				
 
-		void setLUTColor(int rgb) {
+		public void setLUTColor( int rgb ) {
```

Comment:
We have commands to apply similar LUTS and display settings from a JSON file and from an ImageData to all similar images in project. These 3 commands we made here: https://github.com/BIOP/qupath-biop-extensions/tree/dev-0.2.0/src/main/java/ch/epfl/biop/qupath/commands